CPTextView/Frameworks/TextView/CPTypesetter.j

```diff
@@ -399,22 +427,24 @@ var CPSystemTypesetterFactory;
 
             if (isNewline)
             {
-                if ([_currentParagraph minimumLineHeight])
-                    _lineHeight = MAX(_lineHeight, [_currentParagraph minimumLineHeight]);
```

Comment:
this is only executed once for each line. from my understanding, optimizing anything here should not result in any noticeable speed gain.